queries/supercollider/textobjects.scm

```diff
@@ -1,51 +1,60 @@
 ; conditional
 (if
-true:(function_block)? @conditional.inner
-false:(function_block)? @conditional.inner
-) @conditional.outer
+  true: (function_block)? @conditional.inner
+  false: (function_block)? @conditional.inner) @conditional.outer
 
 (if
-(_)? @conditional.inner
-) @conditional.outer
+  (_)? @conditional.inner) @conditional.outer
 
 ; loops
 (for
-(function_block)? @loop.inner) @loop.outer
-(while
-(function_block)? @loop.inner) @loop.outer
-
+  (function_block)? @loop.inner) @loop.outer
 
+(while
+  (function_block)? @loop.inner) @loop.outer
 
 (code_block
-(_) @block.inner
-) @block.outer
+  (_) @block.inner) @block.outer
 
-(class
-) @class.outer
+(class) @class.outer
 
 (parameter_call_list
-(_)? 
-) @class.inner
+  (_)?) @class.inner
 
 (line_comment) @comment.outer
+
 (block_comment) @comment.outer
 
 (function_block
-(_) @function.inner
-) @function.outer
+  (_) @function.inner) @function.outer
 
 ;parameters
 ((parameter_call_list
-"," @_start . (_) @parameter.inner)
-(#make-range! "parameter.outer" @_start @parameter.inner))
-((parameter_call_list
-. (_) @parameter.inner . ","? @_end)
-(#make-range! "parameter.outer" @parameter.inner @_end))
-
+  "," @_start
+  .
+  (_) @parameter.inner)
+  (#make-range! "parameter.outer" @_start @parameter.inner))
 
-(((symbol)?(identifier)
-"," @_start . ((method_name)(function_call(_))) @parameter.inner)
-(#make-range! "parameter.outer" @_start @parameter.inner))
-(((symbol)?(identifier)
-. ((method_name)(function_call(_)))  @parameter.inner . ","? @_end)
-(#make-range! "parameter.outer" @parameter.inner @_end))
+((parameter_call_list
+  .
+  (_) @parameter.inner
+  .
+  ","? @_end)
+  (#make-range! "parameter.outer" @parameter.inner @_end))
+
+(((symbol)?
+  (identifier)
+  "," @_start
+  .
+  ((method_name)
+    (function_call
+      (_))) @parameter.inner)
+  (#make-range! "parameter.outer" @_start @parameter.inner))
+
+(((symbol)?
+  (identifier)
+  .
+  ((method_name)
+    (function_call
+      (_))) @parameter.inner.","? @_end)
+  (#make-range! "parameter.outer" @parameter.inner @_end))
```

Comment:
It trips again ahhh

Can you manually reformat this specific query? It looks like a certain amount of parens can be kicked

Reply:
When running the new formatting script, this ends up as

```scheme
(((symbol)?
  (identifier)
  .
  ((method_name)
    (function_call
      (_))) @parameter.inner.
  ","? @_end)
  (#make-range! "parameter.outer" @parameter.inner @_end))
```

Which gives an error for the Neovim query diagnostics on nightly because `@parameter.inner.` is interpreted as the name of the capture (instead of the last `.` being interpreted as an anchor). Is this intended?

Reply:
It's not intended for sure.

Reply:
Ok it's my fault :)

Please run again.

Reply:
I think I fixed it now (for real). Please drop the formatting commits again and re-format the whole thing :bowing_man: